packages/addresses/src/public-key.ts

```diff
@@ -21,3 +21,20 @@ export async function getAddressFromPublicKey(publicKey: CryptoKey): Promise<Add
     const publicKeyBytes = await crypto.subtle.exportKey('raw', publicKey);
     return getAddressDecoder().decode(new Uint8Array(publicKeyBytes));
 }
+
+/**
+ * Given an {@link Address}, return a {@link CryptoKey} that can be used to verify signatures.
+ *
+ * @example
+ * ```ts
+ * import { getAddressFromPublicKey } from '@solana/addresses';
+ *
+ * const publicKey = await getPublicKeyFromAddress(address);
+ * ```
+ */
+export async function getPublicKeyFromAddress(address: Address, extractable: boolean = false) {
+    assertIsAddress(address);
```

Comment:
Nit: we shouldn't need this because you're already correctly using `Address` as the parameter type.

Reply:
Facts, but now you're making me think we need to assert that Ed25519 importing is supported (ie. we need to add something to `@solana/assertions` to do that)